src/viam/sdk/config/viam-cpp-sdk-libviamsdk.pc.in

```diff
@@ -2,11 +2,11 @@ prefix=@CMAKE_INSTALL_PREFIX@
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
-Name: @PROJECT_NAME@-libviamcpp
+Name: @PROJECT_NAME@-libviamsdk
 Description: @PROJECT_DESCRIPTION@
 URL: @PROJECT_HOMEPAGE_URL@
 Version: @PROJECT_VERSION@
-Requires: gRPC++ >= @VIAMCPPSDK_GRPCXX_VERSION@ protobuf >= @VIAMCPPSDK_PROTOBUF_VERSION@
-Libs: -L${libdir} -L@Boost_LIBRARY_DIRS@ -lviamcpp -lviam_rust_utils
+Requires: gRPC++ >= @VIAMCPPSDK_GRPCXX_VERSION@ protobuf >= @VIAMCPPSDK_PROTOBUF_VERSION@ @PROJECT_NAME@-libviamapi >= @PROJECT_VERSION@
```

Comment:
The `libviamsdk` `pc.in` file now declares a dependency on `libviamapi` so that we automatically pick up the right include paths and such for that.

Reply:
And, having said that, I realized that it probably means we can remove the manual setting for the install-time target include directories for `libviamapi` and `libviamsdk` because the `FILE_SET` takes care of that too. Doing that.